Formula/git-absorb.rb

```diff
@@ -16,6 +16,10 @@ class GitAbsorb < Formula
 
   def install
     system "cargo", "install", "--root", prefix, "--path", "."
+
+    man_path = File.join(prefix, "share", "man", "man1")
+    mkdir_p man_path
+    cp "./Documentation/git-absorb.1", man_path
```

Comment:
Documentation should be installed into `prefix` by `cargo install`. If upstream does not install it by default, we'll leave it out. (Otherwise the manual installation of files tends to bitrot…)

Reply:
Unfortunately, cargo does not yet support installing man files. See https://github.com/rust-lang/cargo/issues/2729. This is an attempt to work around that limitation since there’s no other option that works with rust built-in.

The project maintainer made it clear in https://github.com/tummychow/git-absorb/issues/2#issuecomment-438525999 that he considers system installation to be out of scope for the project. 
> I'm way too lazy to package git-absorb for system package managers, so getting the manpage into the right place in /usr/share/man is gonna have to be someone else's problem.

I’m hoping that we can make this man page installation (and therefore git help) 
work for this project despite rust/cargo‘s current lack of support.

Is this leaving out documentation not automatically installed a hard requirement? I hope not, but if so, I’ll close this PR. 

For my understanding, does the bitrot that you’ve seen occur come because the source of the documentation isn’t included in the package file? Because its location or format changes? Because it’s method of building changes?

Reply:
> because the source of the documentation isn’t included in the package file? Because its location or format changes? Because it’s method of building changes?

We've seen all three. Because new man page is created, also.

> getting the manpage into the right place in /usr/share/man is gonna have to be someone else's problem

Lovely.

Reply:
The concern you raise is legitimate. To protect against those scenarios, I can add formula tests that assert that the set of man files under `Documentation/` matches the set of man files installed to `share/man/`. Then if anything changes that causes it not to be installed, it will become immediately apparent when updating the formula.